### PR TITLE
Add `comparison` concept

### PR DESCRIPTION
**Pull request by andrerfcsantos**
Part of #1611

### CODE REVIEW

File: concepts/comparison/about.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# About
+
+In Go, we can compare values using the equality and ordering operators. We have 6 of these operators:
+
+| Comparison        | Operator  |
+| ------------------| --------- |
+| Equal             | `==`      |
+| Not Equal         | `!=`      |
```
Comment (jmrunkle) [inline concepts/comparison/about.md:8]:
```suggestion
| equal             | `==`      |
| not equal         | `!=`      |
```

Consistent capitazilation.

File: concepts/comparison/about.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# About
+
+In Go, we can compare values using the equality and ordering operators. We have 6 of these operators:
```
Comment (jmrunkle) [inline concepts/comparison/about.md:3]:
```suggestion
In Go, we can compare values using the equality and ordering operators.
We have 6 of these operators:
```

nit: one sentence per line

File: concepts/comparison/about.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```go
+a := 3
+
+a != 4 // true
+a > 5  // false
+```
+
+## Equality operators
+
+The equality operators `==` and `!=` check whether a value is equal to another or not, respectively. We can use the equality operators on the majority of types in Go. Here are some common exmaples:
```
Comment (jmrunkle) [inline concepts/comparison/about.md:25]:
```suggestion
The equality operators `==` and `!=` check whether a value is equal to another or not, respectively.
We can use the equality operators on the majority of types in Go. Here are some common exmaples:
```

File: concepts/comparison/about.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+2.1 != 2.2 // true, float comparison 
+
+"hello" == "hello" // true, string comparison
+
+'a' != 'b' // true, rune/byte comparison
+```
+
+## Ordering operators
+
+Ordering operators check if one value is greater than (`>`), greater or equal to (`>=`), less than (`<`) and less or equal to (`<=`) to another value. This kind of comparison is available for numbers, bytes, runes and strings. When comparing strings, the dictionary order (also known as lexicographic order) is followed.
```
Comment (jmrunkle) [inline concepts/comparison/about.md:39]:
```suggestion
Ordering operators check if one value is greater than (`>`), greater or equal to (`>=`), less than (`<`) and less or equal to (`<=`) to another value.
This kind of comparison is available for numbers, bytes, runes and strings.
When comparing strings, the dictionary order (also known as lexicographic order) is followed.
```

**Review by jmrunkle — APPROVED** [review]
Mostly looks good, just some very nitpicky comments. Thanks!

**Comment by andrerfcsantos** [comment]
Thanks for the review!

Suggestions applied to both files.

**Pull request merged**